d2cli/main.go

```diff
@@ -285,11 +285,33 @@ func compile(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, sketc
 	if sketch {
 		opts.FontFamily = go2.Pointer(d2fonts.HandDrawn)
 	}
+
+	t := time.NewTicker(time.Second * 5)
+	defer t.Stop()
+	stop := make(chan struct{}, 1)
+	defer func() {
+		stop <- struct{}{}
+	}()
+
+	go func() {
+		for {
+			select {
+			case <-t.C:
+				ms.Log.Info.Printf("compiling & running layout algorithms...")
+			case <-stop:
+				ms.Log.Info.Printf("done")
```

Comment:
Is this necessary?

Reply:
oops, thx for catching, it was just to debug that it actually stops